### DISCUSSION TITLE
评论区头像问题

### DISCUSSION THREAD

**Discussion by kannafay** [Tokinx/Wing#44] [讨论]
更换成 https://cravatar.cn/avatar/ 地址后评论区头像源地址依然没被替换
![image](https://user-images.githubusercontent.com/85301126/189495753-4b93c760-1edd-4089-b391-2f6ef0105536.png)
![image](https://user-images.githubusercontent.com/85301126/189495725-0f521222-c7b2-47b5-bc45-d89c5e06013e.png)

**Comment by Tokinx** [comment]
thanks, 这个问题将在v0.7.21版本中修复

**Reply by moenyi** [reply]
> thanks, 这个问题将在v0.7.21版本中修复

在访客状态下，未输入邮箱的情况下图像还是碎裂状态

**Reply by Tokinx** [reply]
你的静态文件似乎被缓存了，helper.js刷新一下缓存即可
